pkg/deploy/docker_test.go

```diff
@@ -33,6 +34,42 @@ func Test_parseCmd(t *testing.T) {
 }
 
 func Test_runShellCmd(t *testing.T) {
-	err := runShellCmd(os.Stdout, os.Stderr, "echo", "hello")
-	assert.Nil(t, err)
+
+	t.Run("should return the command output in output buffer", func(t *testing.T) {
+		var outBuf bytes.Buffer
+		err := runShellCmd(&outBuf, os.Stderr, "echo", "hello")
+		assert.Nil(t, err)
+		assert.Equal(t, "hello\n", outBuf.String())
+	})
+
+	t.Run("should return in error buffer all the contents of stderr", func(t *testing.T) {
+		var errBuf bytes.Buffer
+		errSecond := runShellCmd(os.Stdout, &errBuf, "/bin/bash", "-c", "echo hello err 1>&2")
```

Comment:
Thanks for the changes, they look good @ABrunoRaise. 

One last thing: the new shell tests for the error case use bash directly. They will only work on linux with bash installed. On windows they will fail (and macos too I think as it uses zsh by default). Can you try to change to just /bin/sh and see if they still work as before? At least that should cover macos.

Reply:
Thanks for the feedback, I will test the change.